packages/pipeline/README.md

```diff
@@ -45,6 +45,27 @@ environment variable `ZEROEX_DATA_PIPELINE_DB_URL` which should be a properly
 formatted
 [PostgreSQL connection url](https://stackoverflow.com/questions/3582552/postgresql-connection-url).
 
+## Test environment
```

Comment:
Is it cleaner to have containerized test environments? should we have simple instructions for a docker psql server?

Reply:
👍 I added Docker instructions as the preferred method. I also mentioned that there are other possible ways of installing Postgres that would work fine.